uswds/uswds-site
# USWDS-Site: Add known issues section to date picker page

# Summary

Added a known issues section to the date picker component page.

## Related PR
This is a partial implementation of the prototype found in https://github.com/uswds/uswds-site/pull/2326

## Preview link

Preview link: [Date picker page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-add-known-issues/components/date-picker/)

## Testing and review

- Confirm a known issues section appears on the date picker page
- Confirm the info alert is present at the top of the date picker page
- Confirm that there is no known issues section or alert on other component pages
- On known issues list items:
  - Confirm that the issue descriptions are accurate
  - Confirm that GitHub links work as expected
- Confirm there are no spelling or grammar issues
